api/k8s/v1/k8s.proto

```diff
@@ -1141,3 +1157,45 @@ message RemoveObjectMetaFields {
   repeated string labels = 1 [ (validate.rules).repeated = {unique : true} ];
   repeated string annotations = 2 [ (validate.rules).repeated = {unique : true} ];
 }
+
+message Node {
+  option (clutch.api.v1.id).patterns = {
+    type_url : "clutch.k8s.v1.Node",
+    pattern : "{cluster}/{name}"
+  };
+  string name = 1;
+  string cluster = 2;
+  bool unschedulable = 3;
```

Comment:
Would it make sense for this be it's own message for Taint specific values?

Reply:
cordon's not a taint, it is a separate field on the NodeSpec

Reply:
ah I see it now, thanks!